cmd/notary/keys.go

```diff
@@ -10,37 +11,81 @@ import (
 	"time"
 
 	"github.com/docker/notary/keystoremanager"
+	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/trustmanager"
 
 	"github.com/spf13/cobra"
 )
 
 func init() {
-	cmdKeys.AddCommand(cmdKeysRemoveRootKey)
-	cmdKeys.AddCommand(cmdKeysGenerateRootKey)
+	cmdKey.AddCommand(cmdKeyList)
+	cmdKey.AddCommand(cmdKeyRemoveRootKey)
+	cmdKey.AddCommand(cmdKeyGenerateRootKey)
+
+	cmdKeyExport.Flags().StringVarP(&keysExportGUN, "gun", "g", "", "Globally unique name to export keys for. A new password will be set for all the keys. Output format is a zip archive.")
```

Comment:
Hmm, I'm not sure I follow. If we made those changes, would the supported forms be:

    notary key export docker.com/docker file.zip
    notary key export --all file.zip

I think that when we have positional arguments, we should make them consistent no matter what flags are specified. But maybe I'm misunderstanding your suggestion.